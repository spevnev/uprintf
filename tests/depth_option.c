#define UPRINTF_MAX_DEPTH 3
#include "linked_list.c"
