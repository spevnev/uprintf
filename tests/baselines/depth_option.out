Linked list: {
    int value = 0
    Node *next = POINTER ({
        int value = 2
        Node *next = POINTER ({
            int value = 4
            Node *next = POINTER ({...})
        })
    })
}
